recipes/dgenies/post-link.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sed -i.bak \
```

Comment:
What does this do? You can do that in a build.sh file, during package build and not during installation time, I guess.
Maybe it would also be good to only do that for OSX - if I got the PR description correct?